eboxbw/cli.py

```diff
@@ -157,6 +164,48 @@ def print_table_header():
 
         print_table_border()
 
+def _print_machine(usage_info, conv_func, punit, details):
+
+    def print_row(date, dl, ul, cb, date_cb):
+        date_txt = date_cb(date)
+        fmt_row1 = '{}::{}::{}::{}'
+
+        print(fmt_row1.format(date_txt, dl.real_gb, ul.real_gb, cb.real_gb))
+
+        if dl.effective_gb is not None:
+            dl_txt = dl.effective_gb
+            ul_txt = ul.effective_gb
+            cb_txt = cb.effective_gb
+
+            print(fmt_row1.format('effective', dl_txt, ul_txt, cb_txt))
+
+    if details:
+        print('{}::{}'.format('plan', usage_info.plan))
+        print('{}::{}'.format('super_off_peak',
+                                         _yes_no_machine(usage_info.has_super_off_peak)))
+        print('{}::{}'.format('extra_blocks',
+                                          usage_info.extra_blocks))
+        print('{}::{}'.format('capacity',
+                                   usage_info.plan_cap.real_gb))
+        print('{}::{}'.format('available_usage',
+                                   usage_info.available_usage.real_gb))
```

Comment:
For all the `print()` above:

* Use CSV format.
* Use hyphens instead of underscores, e.g. `super-off-peak`.
* Follow PEP 8 to align parameters.
* Use `conv_func`.